bpf/bpf_lxc.c

```diff
@@ -663,7 +663,7 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 		 * for further processing.
 		 */
 		else if (ret == IPSEC_ENDPOINT)
-			goto pass_to_stack;
+			return CTX_ACT_OK;
```

Comment:
We lose the monitor messages with this approach, is that intended?

Reply:
It was intended but agree its not such a good idea. I'll move it around so we don't lose this.